__tests__/utils.tsx

```diff
@@ -0,0 +1,139 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://github.com/serlo/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import {
+  sanitizeHtml,
+  markdownToHtml,
+  ALL_LANGUAGE_CODES,
+  isLanguageCode,
+  PreactResponse,
+  JsonResponse,
+  NotFoundResponse
+} from '../src/utils'
+
+import { h } from 'preact'
+import { Template } from '../src/ui'
+
+test('ALL_LANGUAGE_CODES', () => {
+  expect(ALL_LANGUAGE_CODES.length).toBeGreaterThan(0)
+})
+
+describe('isLanguageCode()', () => {
+  expect(isLanguageCode('de')).toBe(true)
+  expect(isLanguageCode('fr')).toBe(true)
+
+  expect(isLanguageCode('serlo')).toBe(false)
+  expect(isLanguageCode('EN_EN')).toBe(false)
+  expect(isLanguageCode('')).toBe(false)
+})
```

Comment:
Not needed with suggested change below

Reply:
I need a type guard at https://github.com/serlo/serlo.org-cloudflare-worker/blob/f503df6aba7b6d3c1a549be0c8b6888fb4f3dc74/src/static-pages/index.tsx#L78-L81 However we might implement https://stackoverflow.com/questions/58278652/generic-enum-type-guard/58278753#58278753